.circleci/config.yml

```
@@ -536,6 +658,20 @@ workflows:
             - android_test
             - coverage
             - build_ios
+            - docs_release
+
+      - build_docs:
+          name: build-docs-main
+          context:
+            - Publish
+          requires:
+            - bazelrc
+
+      - docs_release:
+          context:
+            - Publish
+          requires:
```

IMO this should run in parallel with when we do the package releases.

---

If its parallel it could fail and the rest of the stages would still execute (unless we set it to fail fast)

---

I had pushed for a pre-requisite to ensure we _can_ publish docs before releasing.

> We did which is why we had it last in the script. I don't think a doc release should fail the overall release because its easier for us to run manually.

At least this change won't affect releases, which was the issue — docs publishing failed after all the packages were published. But it prevented the tag from being written and released to GH, requiring a re-run with all my fingers crossed.

Going w/ a parallel deploy makes sense to me (w/o fail fast to avoid cancelling artifact publishing halfway through). Then we can just run the command locally if it does fail.